openapi/definitions.yaml

```diff
@@ -1355,6 +1364,34 @@ ImpersonatedService:
     - service_id
     - user_groups
     - user_email
+ThirdPartyMessage:
+  type: object
+  properties:
```

Comment:
We need a generic property that will contain the "flexible" part of the third-party message (the one matching the confiugred json schema).

Reply:
I thought it could be a not-strinct implementation, so extended properties could be handled without the need of spec definition.
Do you think there is something wrong with it? 
cc @AleDore

Reply:
In [fb1e012](https://github.com/pagopa/io-functions-commons/pull/275/commits/fb1e0127014af737e69d324886ae90d23bbea584) I've added `additionalProperties: true` to ThirdPartyMessage, so we expect free-form extra data to be present.

This will allow ThirdParty services to include extra properties that will be included in decode even with `t.exact` decoder.